Libs/MRML/Core/vtkMRMLDisplayableNode.cxx

```diff
@@ -319,8 +319,9 @@ void vtkMRMLDisplayableNode::SetDisplayClassVisibility(const char* nodeClass, in
 }
 
 //---------------------------------------------------------------------------
-void vtkMRMLDisplayableNode::GetRASBounds(double bounds[6])
+bool vtkMRMLDisplayableNode
+::GetRASBounds(double bounds[6], bool useTransform)
```

Comment:
I agree for the return value.
Yes, if extensions rely on this method it might be too much burden for them to change it. I'll add a method with a different name then.